pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -9948,6 +9949,17 @@ spec:
 	reconciledRun, _ := prt.reconcileRun("foo", "test-pipelinerun", []string{}, false)
 
 	checkPipelineRunConditionStatusAndReason(t, reconciledRun, corev1.ConditionUnknown, v1beta1.PipelineRunReasonRunning.String())
+
+	trustedResourcesCodition := reconciledRun.Status.GetCondition(trustedresources.ConditionTrustedResourcesVerified)
+	wantCondition := &apis.Condition{
+		Type:    trustedresources.ConditionTrustedResourcesVerified,
+		Status:  corev1.ConditionTrue,
+		Reason:  trustedresources.ReasonResourceVerificationSucceeded,
+		Message: trustedresources.MessagePassVerification,
+	}
+	if d := cmp.Diff(wantCondition, trustedResourcesCodition, cmpopts.IgnoreFields(apis.Condition{}, "LastTransitionTime")); d != "" {
```

Comment:
nit: suggest creating a variable for cmpopts.IgnoreFields(apis.Condition{}, "LastTransitionTime")